db/migrate/20150618183029_add_extensions_to_versions.rb

```diff
@@ -0,0 +1,5 @@
+class AddExtensionsToVersions < ActiveRecord::Migration
+  def change
+    add_column :versions, :extensions, :boolean
```

Comment:
To be honest, this was done on purpose. The thing is, this data can be received only on gemspec push (because it was not previously extracted from it). That's why there is 3 states: Unknown (when we didn't process the gemspec, so we can't say for sure is there something or not), Yes (we know that there is something in extensions array), No (extensions array is empty).

Am I wrong in assumptions? Because re-processing millions of gems (unpack, get gemspec) is not fast operation.
In approach I described, we will update this info only for gems which is active or will ship version exactly for this.